lib/cloudant.js

```diff
@@ -712,7 +706,10 @@ Cloudant.prototype.updateAttributes = function(model, id, data, options, cb) {
     _.mergeWith(doc, data, function(dest, src) { return src; });
     self.create(model, doc, options, function(err) {
       if (err) return cb(err);
-      return cb(err, data);
+      mo.db.get(id, function(err, doc) {
```

Comment:
@ssh24 regardless what juggler does with the data, we can still callback the full `result` in `_insert`, it's a connector private function and won't be used for juggler.

If `Cloudant.prototype.create()` expect only `id` returned, then let itself do the result filter, and here in `updateAttributes()`, we can call `self._insert` instead of `self.create`, unless there are required check in `create()`